timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -67,49 +155,52 @@ class AbstractTimeSeriesModel(AbstractModel):
         If None, model defaults are used. This is identical to passing an empty dictionary.
     """
 
+    model_file_name = "model.pkl"
+    model_info_name = "info.pkl"
     _oof_filename = "oof.pkl"
+
     # TODO: For which models should we override this parameter?
     _covariate_regressor_fit_time_fraction: float = 0.5
     default_max_time_limit_ratio: float = 0.9
 
-    # TODO: This is a hack to override the AbstractModel method, which the HPO module
-    # also circumvents with an ugly None-check.
-    estimate_memory_usage: Callable = None  # type: ignore
-
     _supports_known_covariates: bool = False
     _supports_past_covariates: bool = False
     _supports_static_features: bool = False
 
     def __init__(
         self,
-        freq: Optional[str] = None,
-        prediction_length: int = 1,
         path: Optional[str] = None,
         name: Optional[str] = None,
+        hyperparameters: Optional[Dict[str, Any]] = None,
+        freq: Optional[str] = None,
+        prediction_length: int = 1,
         metadata: Optional[CovariateMetadata] = None,
+        target: str = "target",
+        quantile_levels: List[float] = [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9],
```

Comment:
Mutable default values can be dangerous?

Reply:
Yes, thanks for flagging. Changing to 
```
quantile_levels: Sequence[float] = (0.1, ..., 0.9)
```

and coercing to list inside. the constructor.